internal/control_test_engine/gnb/ngap/handler.go

```diff
@@ -706,7 +713,7 @@ func HandlerAmfConfigurationUpdate(amf *context.GNBAmf, gnb *context.GNBContext,
 			for _, toAddItem := range toAddList.List {
 				bitLen := toAddItem.AMFTNLAssociationAddress.EndpointIPAddress.Value.BitLength
 				var ipv4String string
-				if bitLen == 32 || bitLen == 160 {
+				if bitLen == 32 || bitLen == 160 { // IPv4 or IPv4+IPv6
 					ipv4String = bytesToIpv4String(toAddItem.AMFTNLAssociationAddress.EndpointIPAddress.Value.Bytes)
```

Comment:
Would ngapConvert.IPAddressToString work instead of creating a new function?

Reply:
Yes !